Sources/Edge.swift

```diff
@@ -238,4 +241,25 @@ public class Edge: NSObject, Extension {
         return state?.implementationDetails
     }
 
+    /// Set the location hint for the Edge Network. The new location hint and expiry date (calculated from the ttlSeconds) are updated in memory and in
+    /// persistent storage. If the new location hint is different from the previous, then a shared state is also created with the new hint.
+    /// A nil `hint` value will clear the location hint in memory, persistent storage, and create a new shared state if a previous location hint was set.
+    /// - Parameters:
+    ///   - hint: the new EdgeNetwork location hint to set
+    ///   - ttlSeconds: the time-to-live for the location hint
+    private func setLocationHint(hint: String?, ttlSeconds: TimeInterval?) {
+        guard let state = state else { return }
+        if let hint = hint, let ttlSeconds = ttlSeconds {
+            state.setLocationHint(hint: hint, ttlSeconds: ttlSeconds, createSharedState: createSharedState(data:event:))
```

Comment:
Do other extensions use the locationHint from the shared state?

Reply:
Not currently and it's not expected as they should send requests through the Edge extension. The shared state is intended for Rules use cases where a customer may want to send a postback directly to the Edge Network. There's no hard requirement for that use-case either, however.